src/services/porssisahko.rs

```diff
@@ -0,0 +1,83 @@
+use std::u32;
+
+use anyhow::Result;
+use cached::proc_macro::cached;
+use chrono::{DateTime, Utc};
+use plotters::{chart::{ChartBuilder, LabelAreaPosition}, prelude::{BitMapBackend, IntoDrawingArea}, series::LineSeries, style::{BLUE, WHITE}};
+use serde::Deserialize;
+
+#[cached(result = true, time = 60)]
+pub async fn get_price_chart() -> Result<Vec<u8>> {
```

Comment:
Should this be in the service itself, or should it be a part of the command file?

I suppose it could make more sense to keep the REST API consumption only in the service, and constructing the TG message/content in the command file?